src/content/resources/concepts/layer/overview.md

```diff
@@ -234,4 +234,4 @@ This allows developers to use their preferred format. A tool designed for one fo
 
 Currently, converters are available to convert XML-based OSCAL files to JSON and JSON-based OSCAL files to XML.
 
-For more details on converter usage, see the build and usage instructions on the [tools page](../../tools/_index.md#data-conversion).
+For more details on converter usage, see the build and usage instructions on the [tools page](https://pages.nist.gov/OSCAL/resources/tools/#data-conversion).
```

Comment:
I suggest keeping the relative path to the pages created by OSCAL-Pages, because when testing locally and building the site you either point to the existing link on the website which might be unavailable /non-existing . The link correction should be (../../tools/#data-conversion) instead of (https://pages.nist.gov/OSCAL/resources/tools/#data-conversion)

Reply:
I see. Thank you for the suggestion! I will try to work it out in the file.

Reply:
Hello Michaela, I tried to use the link correction you suggested: (../../tools/#data-conversion) and when I build the site locally, the link leads to a 404 error. However, when I use the previous link: (https://pages.nist.gov/OSCAL/resources/tools/#data-conversion), there is no error in access the "tools page". 

Is it possible to utilize the absolute path instead of the relative path? It seems on my end the relative path is leading to a 404 error page.

Reply:
@mlynnguyen  - IF the site gets build correctly, and the relative path is valid, then the relative link should work. When you include the full path (hardcoding it) the link points always to an existing page online and not to one you are building locally when you test the build.  I'll take a look and test locally myself.